rotkehlchen/exchanges/binance.py

```diff
@@ -285,14 +287,29 @@ def api_query(self, method: str, options: Optional[Dict] = None) -> Union[List,
                         code,
                         msg,
                     ))
-            elif response.status_code == 429:
-                if backoff > self.backoff_limit:
-                    break
-                # Binance has limits and if we hit them we should backoff
-                # https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md#limits
-                log.debug('Got 429 from Binance. Backing off', seconds=backoff)
-                gevent.sleep(backoff)
-                backoff = backoff * 2
+            elif response.status_code in (418, 429):
```

Comment:
@vnavascues A change I made here is to also handle 418 since according to the docs the behaviour is the same.